omlmd/helpers.py

```diff
@@ -27,20 +26,18 @@ def download_file(uri: str):
     return file_name
 
 
+@dataclass
 class Helper:
-    _listeners: list[Listener] = []
-
-    def __init__(self, registry: OMLMDRegistry | None = None):
-        if registry is None:
-            self._registry = OMLMDRegistry(
-                insecure=True
-            )  # TODO: this is a bit limiting when used from CLI, to be refactored
-        else:
-            self._registry = registry
+    registry: OMLMDRegistry = (
```

Comment:
I figured there was no point in making this private if it can be user-provided

Reply:
frankly I preferred it private because I wanted to make sure it's not "changeable" in between operations on a Helper instance; imho. Wdyt ?